docs/DeveloperGuide.md

```diff
@@ -302,29 +302,62 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: Add a person**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to add person
+2.  AddressBook adds the person
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. Given contact has invalid or duplicate fields.
+    * 1a1. AddressBook shows an error message.
+
```

Comment:
Did we say that the address book will prompt the user that if they really want a duplicate entry, they need to write a nickname?